roles-in-tech/technical-program-manager.md

```diff
@@ -2,89 +2,120 @@
-- More things
-- Even more things
-- Even more things
+**Technical Program Manager (TPM) is Responsible for:**
 
+- Enabling the team to launch various features
+- Leads the sprints, gathers requirements from the product managers
+- Sets up architectural reviews with teams consuming the service
+- Helps out in on-call and escalation
 
-### Common Mistakes / Misconceptions
+**Types of Technical Program Managers**
 
-List things that apprentices might not realize, might assume at first, or should avoid.
+* Front End
+* Backend
+* Integration
+* Business Intelligence
+* ERP
+* Localization
+* Hardware
+* Data Center
+* Process
+* Security
+* Cloud TPM
+* UX/UI
 
-- Example
-- Example
+Perhaps the biggest **advantage** to being a **technical project manager** is having the ability to fill the communication gap between the development team and the business team.  This can manifest itself in many ways:
 
+* Describing how individual development components relate to the big picture for an organization
+* Ensuring the business team understands why certain technical decisions were made along with the 
+  benefits and trade-offs
+* Explaining to the business why a requirement may not be technologically feasible  but here are some alternatives
 
-### Guided Practice
+![tpm](productmanage.png)
 
-Have the apprentices work with you as you do something step-by-step.  This can also be fulfilled by a detailed tutorial intended for beginners.
+**What Now?**
 
+So now you’re convinced that you should become a technical project manager – **but how?**
 
-### Independent Practice
+1. Take Time to Understand Your Project/Product.
+2. Learn to Code on Your Own.
+3. Build Something on your Own.
+
+**Challenges**
 
-Class does this thing themselves with specific additional items. This could be alone, with a partner, or small group; but the idea is that it's less guided, more independent.
+* Lack of loyalty in employees and corporate bosses.
+* Evolution of Technology.
+* Globalism in HR.
+* Self Image and Proper Restraint.
 
-### Challenge
+### Common Mistakes / Misconceptions
+
+* **Tech Project Manager is only skilled in project management**
+* **Tech Project Manager Job is to create reports/ Gantt charts**
+* **No role for Tech Project Manager in Agile context**
+* **Tech Project Manager role is the same across the organisation**
 
-Apprentices can try to do this other thing. Ideally, they will be challenged to connect what they've learned to some previous knowledge or additional research.
+### Independent Practice
 
+Try to manage or lead a project in your current organization or your team.
 
 ### Check for Understanding
 
-Some ideas: have apprentices summarize to each other, make a cheat sheet, take a quiz, do an assignment, or something else that helps them self-assess their understanding. This exercise should help apprentices determine whether they've met the outline objectives, or if they need to review.
+* Are you able to manage or lead the project.
```

Comment:
Are these questions? Should be accompanied by question marks

Reply:
done